tests/test_backend.py

```diff
@@ -3010,6 +3010,12 @@ def test_hashtable_lookup(self):
         self._run_test_case([lookup_results.name], {_INPUT: query})
         os.remove(filnm)
 
+    @check_opset_min_version(11)
+    def test_MatrixDiagPart(self):
+        input_val = np.array([[[1, 2, 3], [4, 5, 6]]], dtype=np.int64)
```

Comment:
Can this also include  tests for matrices of slightly higher ranks (e.g.  rank =3 and rank = 4)?